.github/workflows/reusable-test.yaml

```diff
@@ -113,3 +113,23 @@ jobs:
         uses: actions/upload-artifact@v3
         with:
           path: ./logs
+
+  codespell:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout the repository
+        uses: actions/checkout@v4
+
+      - name: pip cache
+        uses: actions/cache@v4
+        with:
+          path: ~/.cache/pip
+          key: ${{ runner.os }}-pip-${{ hashFiles('**/*.py') }}
+          restore-keys: ${{ runner.os }}-pip-
+
+      - name: Install Codespell
+        run: sudo pip install codespell==2.2.5
+
+      - name: Spell Check
+        run: codespell --config=./.codespell/.codespellrc
```

Comment:
What I meant in my first was not to use the python version at all - use the node version so that we don't introduce new language to the stack please